Modix.Data/Repositories/InfractionRepository.cs

```diff
@@ -292,6 +294,22 @@ public async Task<bool> TryDeleteAsync(long infractionId, ulong deletedById)
             return true;
         }
 
+        public async Task<bool> TryUpdateAync(long infractionId, string newReason)
+        {
+            var entity = await ModixContext.Infractions
+                .Where(x => x.Id == infractionId)
+                .FirstOrDefaultAsync();
+
+            if (entity == null)
+                return false;
+
+            entity.Reason = newReason;
```

Comment:
You might meet some resistance here, because the proper "MODiX way":tm: to do this is probably to create a new infraction, mark the old one as deleted, and add an audit record for the update.

But that's a lot more work, probably requires a schema update, and we can always do that in the future if we really want. I'm personally fine with it as is, but I'll let others weigh in.

Reply:
cc @jrusbatch @jmazouri and @Cisien as other staff. Updates to infractions are fairly locked down. I don't think we require a massive audit trail.